[RDSC-5016] Improving documentation for using full row format and mapping (#3046)
redis/docs: content/integrate/redis-data-integration/data-pipelines/transform-examples/redis-row-format.md

          expression: concat([after.CITY, ', ', after.STATE])
          language: jmespath
    ```

> - When using the `full` row format, all data fields in transformation and key calculations should be prepended with `before` or `after.` depending on whether you want to access the previous or current value of the row. Exception from this case is the `mapping` section where only the `after` values are available. And you should use directly the column/value name without the `after.` prefix.

Reviewer: ```suggestion
- When using the `full` row format, you should prepend all data fields in transformation and key calculations with `before` or `after` depending on whether you want to access the previous or current value of the row. The only exception to this is the `mapping` section where only the `after` values are available, so you should use the column/value name without the `after` prefix.
```
